app/resources/css-default.template

```diff
@@ -282,38 +280,48 @@ We don't use "display: table" because IE 6/7 doesn't support it. IE 6/7 is
 over 10% in penetration ratio.
 */
 
-.fields-table {
-  padding: 4px;
+.card {
```

Comment:
Can you also update the comment at line 265 to say "Cards" instead of "Tables"? Because you updated the class name to "card" too.